From d01d2dea1875a302e73a37a23bccf53114fd4133 Mon Sep 17 00:00:00 2001
From: Lorenzo Pirritano <lpirritano@anaconda.com>
Date: Tue, 25 Jun 2024 14:36:30 +0200
Subject: [PATCH 2/2] fix opt_dep attribute check

---
 alibi_detect/tests/conftest.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/alibi_detect/tests/conftest.py b/alibi_detect/tests/conftest.py
index 9de12f4..1e777c3 100644
--- a/alibi_detect/tests/conftest.py
+++ b/alibi_detect/tests/conftest.py
@@ -12,6 +12,8 @@ def opt_dep(request):
     is used in CI to indicate the optional dependencies installed in the tox environments the tests are run in. See
     setup.cfg and .github/workflows/ci.yml for more details.
     """
+    if not hasattr(request.config.option, "opt_dep"):
+        pytest.skip()
     opt_dep_value = request.config.option.opt_dep
     if opt_dep_value is None:
         pytest.skip()
-- 
2.39.1

